src/mvAppItem.cpp

```diff
@@ -3803,12 +3847,41 @@ DearPyGui::GetEntityParser(mvAppItemType type)
         args.push_back({ mvPyDataType::DoubleList, "default_value", mvArgType::KEYWORD_ARG, "(0.0, 0.0)" });
         args.push_back({ mvPyDataType::IntList, "color", mvArgType::KEYWORD_ARG, "(0, 0, 0, -255)" });
         args.push_back({ mvPyDataType::Float, "thickness", mvArgType::KEYWORD_ARG, "1.0" });
-        args.push_back({ mvPyDataType::Bool, "show_label", mvArgType::KEYWORD_ARG, "True" });
```

Comment:
`show_label` - Here's how we can retain this argument, as well as the old behavior of labels on drag items:

https://github.com/v-ein/DearPyGui-fixes/compare/c62d78d3c60efa041e8a00e8ef41910af2ed441c...feature/drag-show-label

If anyone needs the label displayed differently, there's always `add_axis_tag` and `add_plot_annotation` out there, which can be tied to the drag object using callbacks.

Reply:
Perfect thanks! Also, I'm implementing your patches applying `patch` or copy-pasting, I hope it's not a problem (I don't know if there are easier ways to merge them in my branch)

Reply:
Oh I just tried to implement it but it's complaining about the paramter of `ImPlot::IsColorAuto` the requires a `ImVec4` so I added `config.color.toVec4()`
Is it okay for you? Do you have other ideas?

Reply:
`ImGui`: commit e391fe2e66eb1c96b1624ae8444dc64c23146ef4 (HEAD, tag: v1.90.6-docking, docking)
`ImPlot`: commit f156599faefe316f7dd20fe6c783bf87c8bb6fd9 (HEAD -> master, origin/master, origin/HEAD)
`gcc`: gcc (Ubuntu 12.3.0-1ubuntu1~22.04) 12.3.0

Actually regarding `gcc`, I used to used the version 11. Now I switched to 12 and the VSCode compiler is not giving the error anymore but when I build it, it gives me error, I'm trying to figure out why, but I'd say to keep `.toVec4()` at this point, no?

Reply:
Yeah, might be easier to keep `.toVect4()` for the moment. It's an insteresting question why the operator doesn't work, but no need to waste time on it now.